index.html

```diff
@@ -29,7 +33,7 @@ <h1 class="col-12 text-center">NinjaVerse</h1>
                     <div class="col-12 d-flex justify-content-center">
                       <a href="/profile/index.html" role="bytton" class="btn btn-primary my-5">Sign in</a>
```

Comment:
change the <a href> to <button type="submit">

Reply:
Yes, this has already been changed to "<button form="login" href="/profile/index.html" role="button" class="btn btn-primary my-5">Sign in</button>". Not sure why the old commit is displayed here.